src/couch_quickjs/src/couch_quickjs_scanner_plugin.erl

```diff
@@ -172,8 +172,9 @@ process_ddoc(#st{} = St, DbName, #doc{} = DDoc0) ->
         true ->
             St1 = start_or_reset_procs(St),
```

Comment:
Why not put `St1` inside the try block?

Reply:
Because we want to return the updated or reset processes from the catch ... expression, but and if we put it inside the try we won't have it available there. We expect to always be able to reset/restart the processes on failure.

Reply:
Ah, right, thank you for the explanation!